lib/rubocop/cop/bundler/insecure_protocol_source.rb

```diff
@@ -58,7 +60,7 @@ def autocorrect(node)
 
         private
 
-        def source_range(node)
+        def range(node)
```

Comment:
Like `Util`, the `RangeHelp` mixin has a `source_range()` method already. I changed this method to `range` simply to avoid confusion.